app/views/cdo/create/confirm-all-details.njk

```diff
@@ -23,10 +23,8 @@
 {% block content %}
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <form method="POST" action="{{ model.formAction }}" novalidate>
-      <h1 class="govuk-heading-l">Confirm all details</h1>
-
-    <form method="post" action="{{ model.formAction }}">
+    <form method="POST" action="{{ model.formAction }}" onsubmit="confirmSubmit.disabled = true; return true;" novalidate>
+      <h1 class="govuk-heading-l">Confirm all details1</h1>
```

Comment:
1 at end of line

Reply:
Good spot. I put that in to prove which page was being edited as the button disable wan't working on my first try. That would have been embarrassing to get pushed through. Thanks for saving me on that one